apisix/plugins/request-id.lua

```diff
@@ -218,7 +219,18 @@ function _M.rewrite(conf, ctx)
     local headers = ngx.req.get_headers()
     local uuid_val
     if not headers[conf.header_name] then
-        uuid_val = get_request_id(conf.algorithm)
+        -- request-id support get from query string
+        if conf.get_from_query then
+            local querystring = ngx.req.get_uri_args()
+            if not querystring[conf.header_name] then
```

Comment:
It's weird to use the `header_name` to indicate the query name.

Reply:
Add an id-name parameter?